contracts/SportMarkets/LiquidityPool/SportAMMLiquidityPool.sol

```diff
@@ -326,7 +336,19 @@ contract SportAMMLiquidityPool is Initializable, ProxyOwned, PausableUpgradeable
                 withdrawalRequested[user] = false;
                 emit Claimed(user, balanceAfterCurRound);
             }
+            usersProcessedInRound = usersProcessedInRound + 1;
```

Comment:
I get it's the safer option, but can be delivered outside the for() loop into
`usersProcessedInRound = endCursor;`

Reply:
yes, I have considered that, should help with the gas costs.